Classes/Task/BatchResizeTask.php

```diff
@@ -266,7 +266,7 @@ public function syslog($message, $severity = \TYPO3\CMS\Core\Messaging\FlashMess
                 $logger->info($message);
                 break;
             case \TYPO3\CMS\Core\Messaging\FlashMessage::OK:
-                $logger->log($message);
+                $logger->log(\Psr\Log\LogLevel::INFO, $message);
```

Comment:
Looks wrong both by reading and double-checking the documentation. `$message` **must** naturally be kept as 1st parameter. I wonder how you stumbled upon this problem, changed it in your code like that and figured out it worked.

Reply:
We are kind of right both, i added a additional change, see below.